metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -236,9 +236,18 @@ def __init__(self, ctx: PipelineContext, config: SnowflakeV2Config):
 
         # For database, schema, tables, views, etc
         self.data_dictionary = SnowflakeDataDictionary()
-
         self.lineage_extractor: Optional[SnowflakeLineageExtractor] = None
+        self.aggregator: Optional[SqlParsingAggregator] = None
+
         if self.config.include_table_lineage:
+            self.aggregator = SqlParsingAggregator(
```

Comment:
I know `generate_usage_statistics=False` and `generate_operations=False` are the default, but it'd be good to set them explicitly for some extra code clarity